skribenten-web/frontend/src/routes/saksnummer_/$saksId/brevvelger/$templateId/-components/endreMottaker/EndreMottaker.tsx

```diff
@@ -0,0 +1,324 @@
+          manuellAdresseValues={properties.manuellAdresseValues}
+          onBekreftNyMottaker={properties.onBekreftNyMottaker}
+          onCloseIntent={properties.onAvbrytClick}
+          onTilbake={(from) => properties.tab.setTab(from)}
+          samhandlerValues={properties.samhandlerValuesMedId}
+        />
+      ) : (
+        <Tabs onChange={(s) => properties.tab.setTab(s as EndreMottakerModalTabs)} value={properties.tab.tab}>
+          <Tabs.List>
+            <Tabs.Tab label="Finn samhandler" value="samhandler" />
+            {/* Vi har enda ikke backend støtte for å legge til manuell adresse. Midlertidig har vi bare fjernet muligheten til å komme inn i formet */}
+            {/* <Tabs.Tab label="Legg til manuelt" value="manuellAdresse" /> */}
+          </Tabs.List>
+          <div
+            css={css`
+              margin-top: 1rem;
+            `}
+          >
+            <Tabs.Panel value="samhandler">
+              <SøkOgVelgSamhandlerForm
+                control={properties.control}
+                onCloseIntent={properties.onAvbrytClick}
+                onFinnSamhandlerSubmit={properties.onFinnSamhandlerSubmit}
+                onSamhandlerValg={(id) => {
+                  properties.setSamhandler(id);
+                  properties.tab.setTab("oppsummering");
+                }}
+              />
+            </Tabs.Panel>
+            <Tabs.Panel value="manuellAdresse">
+              <UtfyllingAvManuellAdresseForm
+                control={properties.control}
+                onCloseIntent={properties.onAvbrytClick}
+                onSubmit={() => properties.tab.setTab("oppsummering")}
+              />
+            </Tabs.Panel>
+          </div>
+        </Tabs>
+      )}
+    </div>
+  );
+};
+
+type SamhandlerValuesMedId = { id: string } & FinnSamhandlerFormData;
+
+const OppsummeringsTab = (properties: {
+  samhandlerValues: Nullable<SamhandlerValuesMedId>;
+  manuellAdresseValues: Nullable<ManuellAdresseUtfyllingFormData>;
+  onBekreftNyMottaker: (id: string | Adresse) => void;
+  onCloseIntent: () => void;
+  onTilbake: (from: "samhandler" | "manuellAdresse") => void;
+}) => {
+  if (!properties.samhandlerValues && !properties.manuellAdresseValues) {
+    throw new Error(
+      "Teknisk feil Forventet at enten adresse for samhandler, eller manuell oppslag er supplert til oppsummeringen",
+    );
+  }
+
+  if (properties.samhandlerValues) {
+    return (
+      <HentOgVisSamhandlerAdresse
+        id={properties.samhandlerValues.id}
+        onBekreftNyMottaker={() => properties.onBekreftNyMottaker(properties.samhandlerValues!.id)}
+        onCloseIntent={properties.onCloseIntent}
+        onTilbakeTilSøk={() => properties.onTilbake("samhandler")}
+        typeMottaker={properties.samhandlerValues!.samhandlerType!}
+      />
+    );
+  }
+
+  if (properties.manuellAdresseValues) {
+    const adresse: Adresse = {
+      ...properties.manuellAdresseValues.adresse,
+      linje3: "",
```

Comment:
Hvorfor settes linje3 her?

Reply:
Vi har 2 typer adresse backend gir oss. 'Adresse' inneholder 3 linjer + en linje for navn. 

Formet som skal fylles ut, har 2 linker + navn. Siden oppsummeringskomponenter burde basere seg på typene som kan komme inn, må vi fylle ut linje 3 her